application/src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/request/RouteRequestTransitDataProviderFilter.java

```diff
@@ -109,13 +120,11 @@ private boolean tripTimesPredicate(TripTimes tripTimes, boolean applyTripTimesFi
       return false;
     }
 
-    if (wheelchairEnabled) {
-      if (
-        wheelchairPreferences.trip().onlyConsiderAccessible() &&
```

Comment:
This is removed: `wheelchairPreferences.trip().onlyConsiderAccessible()` intensional?